src/signals/settings/roles/containers/RoleFormContainer/components/RoleForm/index.js

```diff
@@ -27,6 +27,17 @@ const StyledInput = styled(Input)`
 export const RoleForm = ({ role, permissions, onPatchRole, onSaveRole, readOnly }) => {
   const formRef = useRef(null);
   const { isValid, validate, errors, event } = useFormValidation(formRef);
+  const [rolePermissions, setRolePermissions] = useState(role.permissions);
+
+  const handleChange = useCallback(id => onChangeEvent => {
+    if (!onChangeEvent) return;
```

Comment:
Can there be such an occasion when there is `onChangeEvent` value? If so, can you add a test case for it?

Reply:
Actualy not, good catch. I removed that!